<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuration</class>
 <widget class="QDialog" name="Configuration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="mainWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>891</width>
     <height>501</height>
    </rect>
   </property>
   <widget class="QWidget" name="pickCours_page">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>321</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_43">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>Devise de Contre partie</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="listeContrepartie">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_42">
          <item>
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Devise de base</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="listeBase">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="addDevise_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Ajouter autre devise</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_19">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLineEdit" name="browseFile">
          <property name="text">
           <string>Cherche un fichier</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="browseButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>231</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bitstream Charter</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Choisir l'evolutionCours</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancel_button">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>460</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>360</y>
       <width>170</width>
       <height>122</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCommandLinkButton" name="ModeManule_button">
        <property name="text">
         <string>Mode Manuel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="ModePas_Pas_button">
        <property name="text">
         <string>Mode Pas à pas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="ModeAuto_buton">
        <property name="text">
         <string>Mode Automatique</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>290</y>
       <width>401</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nom de la simulation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameSimulation">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="config_page">
    <widget class="QWidget" name="strategie_widget" native="true">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>220</y>
       <width>471</width>
       <height>231</height>
      </rect>
     </property>
     <widget class="QWidget" name="maParameters" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>471</width>
        <height>131</height>
       </rect>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>261</width>
         <height>28</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>EMA Period</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="pickEmaPeriod"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QComboBox" name="listeStrategie">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Strategie du trading</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>174</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="strategie_label">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="text">
          <string>Parametres du strategie: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rsiParameters" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>471</width>
        <height>131</height>
       </rect>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>471</width>
         <height>92</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>look back period</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="pickLookbackPeriod"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>over bought boundary</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>over sold boundary</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="pickOversold"/>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="pickOverbought"/>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>181</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Paramètres pour le RSI</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>471</width>
       <height>156</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Date debut </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="pickStartDate"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Interet du broker</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="pickPourcentage"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Montant de Devise de Contre partie</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="pickContrepartie"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Montant de Devise de Base</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="pickBase"/>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>460</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="back_button">
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="creatSimul_button">
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>Configuration</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>513</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
